packages/compass-import-export/test/csv/types/double.csv

```diff
@@ -5,9 +5,7 @@ double,notes
 -1.2
 "1.0"
 "1.2"
-1e-15
+1e-15,a very small number close to 0
```

Comment:
Feel free to ignore, but you might want to add a test for some pathological cases like `1e0` here

Reply:
Cool that caught something. Keep them coming ;)